src/content/reference/kubectl-gs/template-cluster.md

```diff
@@ -0,0 +1,190 @@
+---
+title: kubectl gs template cluster
+description: Reference documentation on how to create a manifest for a Cluster using 'kubectl gs'.
+date: 2020-10-02
+type: page
+weight: 10
+---
+
+# `kubectl gs template cluster`
+
+{{% kgs_alias_assumption %}}
+
+This command helps with creating a cluster by producing a manifest based on user input. This manifest can then optionally be modified and finally be applied to the Control Plane Kubernetes API to create a cluster.
+
+The outcome depends on the provider, set via the `--provider` flag:
+
+For AWS (`--provider aws`):
+
+- [`Cluster`](/reference/cp-k8s-api/clusters.cluster.x-k8s.io/) (API version `cluster.x-k8s.io/v1alpha2`) - holds the base cluster specification.
+- [`AWSCluster`](/reference/cp-k8s-api/awsclusters.infrastructure.giantswarm.io/) (API version `infrastructure.giantswarm.io/v1alpha2`) - holds AWS-specific configuration.
+- [`G8sControlPlane`](/reference/cp-k8s-api/g8scontrolplanes.infrastructure.giantswarm.io/) (API version `infrastructure.giantswarm.io/v1alpha2`) - specifies the master nodes
+- [`AWSControlPlane`](/reference/cp-k8s-api/awscontrolplanes.infrastructure.giantswarm.io/) (API version `infrastructure.giantswarm.io/v1alpha2`) - specifies the master nodes with AWS-specific details
+
+For Azure (`--provider azure`):
+
+- [`Cluster`](/reference/cp-k8s-api/clusters.cluster.x-k8s.io/) (API version `cluster.x-k8s.io/v1alpha3`) - holds the base cluster specification.
+- [`AzureCluster`](/reference/cp-k8s-api/azureclusters.infrastructure.cluster.x-k8s.io/) (API version `infrastructure.cluster.x-k8s.io/v1alpha3`) - holds Azure-specific configuration.
+- [`AzureMachine`](/reference/cp-k8s-api/azuremachines.infrastructure.cluster.x-k8s.io/) (API version `infrastructure.cluster.x-k8s.io/v1alpha3`) - specifies the master nodes.
+
+**Note:** The CRs generated won't trigger the creation of any worker nodes. Please see the [template nodepool](/reference/kubectl-gs/template-nodepool/) for instructions on how to create worker node pools.
+
+## Usage
+
+The command to execute is `kubectl gs template cluster`.
+
+It supports the following flags:
+
+- `--provider` - The infrastructure provider (either `aws` or `azure`)
```

Comment:
Consistency with the `template nodepool` command

```suggestion
- `--provider` - The infrastructure provider (either *aws* or *azure*)
```

Reply:
We should use backticks for literals. I'll align the other command then.